### PR TITLE
Shuffle train subset for summarization example

### LINKED ISSUE #3892: ❓ Summarization example : Why no shuffling ?
# ❓ Questions & Help

Usually, when loading using Dataloader, data is shuffled for the training set. But in the case of the summarization example, data is not shuffled for training set :

https://github.com/huggingface/transformers/blob/1dc9b3c7847269961458c059ad8ad443b26bf60d/examples/summarization/bart/finetune.py#L105-L108

---

**Why data is not shuffled ?**

### PR DESCRIPTION
**Pull request by astariul**
Fix #3892 
@sshleifer

### CODE REVIEW

**Review by sshleifer — APPROVED** [review]
LGTM!
